bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/PCMConverter.java

```diff
@@ -94,8 +94,8 @@ private void createFlowBetweenPreviousAndCurrentNode(Node source, Node dest, Abs
         if (source == null || dest == null) {
             return;
         }
-        List<DataFlowVariable> flowVariables = pcmVertex.getAllDataCharacteristics();
-        for (DataFlowVariable flowVariable : flowVariables) {
+        List<DataCharacteristic> flowVariables = pcmVertex.getAllDataCharacteristics();
```

Comment:
flowVariables is still the wrong name.

Reply:
Fixed in 0b42930fbe7c8d12efeda91e100c91d284e66260